README.md

```diff
@@ -3,7 +3,7 @@
         <img src="https://sentry-brand.storage.googleapis.com/sentry-logo-black.png" width="280">
     </a>
 <br/>
-    <h1>Official Sentry SDK for iOS / tvOS / macOS / watchOS <sup>(1)</sup>.</h1>
+    <h1>Official Sentry SDK for iOS / tvOS / macOS / partially watchOS <sup>(1)</sup>.</h1>
```

Comment:
L: what's about using `limited` instead? so it matches the explanation below

Reply:
I also like limited more.